packages/client-api-schema/src/generated-schema.json

```diff
@@ -649,7 +652,7 @@
       ],
       "type": "object"
     },
-    "JsonRpcError<402,\"Invalid app data\",structure-1165839259-943-963-1165839259-875-965-1165839259-853-966-1165839259-0-1291>": {
+    "JsonRpcError<402,\"Invalid app data\",structure-1165839259-920-940-1165839259-852-942-1165839259-830-943-1165839259-0-1233>": {
```

Comment:
What is this structure stuff?

Reply:
My guess is it's references to line numbers or something -- the difference `y-x` in `1165839259-x-y` seems to be constant in most of this line change: it's 20, 20, 23, and increased by 4.

The numbers don't seem related to the lines in the file though ...